### DISCUSSION TITLE
anyway to limit the streams to only the image capture no video for allowed cameras.

### DISCUSSION THREAD

**Discussion by lgkahn** [mrlt8/docker-wyze-bridge#1075] [Q&A]
thanks if anyone needs help i got this working in container station on a qnap nas.. geting the ip address to work was a challenge as it only by default comes on on one of the nap ips which doesnt work..

anyway.. to avoid clobering my network with 23 cameras (i am able to limite cameras) but i want it to not pull in video only the still cam every so many seconds.. i dont see any options to do that.. ? anyone have a solution or ideas>

thanks in advance

**Comment by mrlt8** [comment]
One option would be to manually trigger the snapshots as needed via the api or mqtt. However, we are using the video stream to capture the image, so you may want to space the snapshots apart so that they don't all start at the same time.
